db-persistence/my/SetupDatabase.ps1

```diff
@@ -19,11 +23,25 @@ if ($restartingInstance) {
 
         $sqlConn = new-object Microsoft.SqlServer.Management.Common.ServerConnection
 
-        $toCopy = @()
-
         $smo = new-object Microsoft.SqlServer.Management.SMO.Server($sqlConn)
-        $smo.Databases | ForEach-Object {
-            if ($_.Name -ne 'master' -and $_.Name -ne 'model' -and $_.Name -ne 'msdb' -and $_.Name -ne 'tempdb') {
+        $dbs = New-Object Collections.Generic.List[object]
+        
+        foreach ($db in $smo.Databases) {
+            $dbs.Add($db)
+        }
+        
+        $tenantDb = $dbs | where Name -eq "tenant"
```

Comment:
Wouldn't that be a different name if you restore a multitenant DB? Or is it always "tenant" in that scenario?

Reply:
In mutlitenant containers there seems to be always a "tenant" database being used as the application database and a per-tenant database like Cronus with the tenant specific data.